assets/js/blocks/featured-category/block.js

```diff
@@ -56,6 +59,7 @@ import { withCategory } from '../../hocs';
  * @param {Object} props.overlayColor Overlay color object for content.
  * @param {function(any):any} props.setOverlayColor Setter for overlay color.
  * @param {function(any):any} props.debouncedSpeak Function for delayed speak.
+ * @param {function(any):any} props.triggerUrlUpdate Function to update Shop now button Url.
```

Comment:
If I'm not wrong, this param should have type `function():void` because the function receives no arguments and it returns nothing (`void`):
```suggestion
 * @param {function():void} props.triggerUrlUpdate Function to update Shop now button Url.
```

Reply:
Yes, correct! It should be void.